prometheus/prometheus.spec

```diff
@@ -0,0 +1,123 @@
+# Build with debug info rpm
+%global with_debug 0
+
+%if 0%{?with_debug} > 0
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package %{nil}
+%endif
+
+%global git_commit 625dc242457d7099b2f6e83267822b59f1c4251c
+%global git_shortcommit  %(c=%{git_commit}; echo ${c:0:7})
+
+Name:           prometheus
+Version:        2.7.2
```

Comment:
This seems to be an ancient version. Is that ok?

Reply:
It's newer than what upstream ships, and it's the same version that ships with Openshift 4.1